02-microarray/dimension-reduction_microarray_01_pca.Rmd

```diff
@@ -143,6 +145,8 @@ From here you can customize this analysis example to fit your own scientific que
 
 ***
 
+<!-- Do not delete this line --> <a name="analysis"></a>
```

Comment:
This should fix the anchor link issue. Only putting it here, but will obviously need to be copied to other files.

```suggestion
<!-- Do not delete this line --> <a name="analysis" style="padding-top:56px;margin-top:-56px;">&nbsp;</a>
```

Reply:
Done!